Content.Shared/Materials/Material.cs

```diff
@@ -87,6 +110,27 @@ public void ExposeData(ObjectSerializer serializer)
             serializer.DataField(ref _hardness, "hardness", 1, alwaysWrite: true);
             serializer.DataField(ref _sharpDamage, "sharpdamage", 1, alwaysWrite: true);
             serializer.DataField(ref _bluntDamage, "bluntdamage", 1, alwaysWrite: true);
+            serializer.DataField(ref _icon, "icon", SpriteSpecifier.Invalid, alwaysWrite: true);
+        }
+
+        public static Material GetMaterial(string ID)
```

Comment:
Just remove these methods, honestly. Only one of them is used once and you can just directly use the prototype manager for that one.

Reply:
Done.